composer.json

```diff
@@ -47,15 +47,14 @@
     "composer/ca-bundle": "^1.2",
     "defuse/php-encryption": "^2.2.0",
     "doctrine/annotations": "^1.13.1 || ^2.0",
-    "doctrine/collections": "^1.6.8",
     "doctrine/dbal": "^3.8",
     "doctrine/doctrine-bundle": "^2.6.3",
     "doctrine/doctrine-migrations-bundle": "^3.2.2",
     "doctrine/inflector": "^2.0.2",
     "doctrine/orm": "^2.15",
     "doctrine/persistence": "^3.0.3",
-    "egulias/email-validator": "^3.0.0",
-    "endroid/qr-code": "^4",
+    "egulias/email-validator": "^4.0",
+    "endroid/qr-code": "^5.1",
```

Comment:
I think supporting both versions (also below) would allow a smoother upgrade path for the projects.

Reply:
@jdreesen done